src/spacingData.js

```diff
@@ -1,22 +1,41 @@
+// @flow
 /**
  * Describes spaces between different classes of atoms.
  */
 
-const thinspace = {
+export type Space = {| number: number, unit: "mu" |};
```

Comment:
I think we can use `type {Measurement} from "./units";`.

Reply:
Done.